src/main/resources/application-dev.yml

```diff
@@ -6,7 +6,7 @@ server:
   forward-headers-strategy: native
 spring:
   flyway:
-    enabled: false
+    enabled: true
```

Comment:
Will be interesting to see how this deploys to `dev`, does this mean we've not been running `Flyway`? 🤔

Reply:
Nope! At least not in dev but prod is safe